samcli/commands/deploy/deploy_context.py

```diff
@@ -142,7 +142,9 @@ def run(self):
                 s3_client, self.s3_bucket, self.s3_prefix, self.kms_key_id, self.force_upload, self.no_progressbar
             )
 
-        self.deployer = Deployer(cloudformation_client)
+        self.deployer = Deployer(
+            cloudformation_client, client_sleep=os.getenv("SAM_CLI_POLL_DELAY", str(DEFAULT_CLIENT_SLEEP))
```

Comment:
Could you please move the env variable reading to the deploy command, and pass its value as a parameter to the deploy_context. This will make it easier for us later to refactor this code, and depend on a command options.

Reply:
It's done